src/main/java/pl/pandziak/ticketbookingapp/configuration/DataProvider.java

```diff
@@ -0,0 +1,69 @@
+package pl.pandziak.ticketbookingapp.configuration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import pl.pandziak.ticketbookingapp.domain.screening.ScreeningFacade;
+import pl.pandziak.ticketbookingapp.domain.screening.dtos.ScreeningRequestDto;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Service providing testing data to in-memory database.
+ */
+@Configuration
+class DataProvider {
```

Comment:
dobrze by było uzależnić ten kod od profilu

Reply:
Tak, klasa DataProvider.class powinna być uzależniona od profilu, chociażby "dev". Przeszło mi to przez myśl, ale nie miałam z nimi styczności w praktyce. Gdybym dalej rozwijała tę aplikację, pewnie to by był jeden z tematów w które bym się zagłębiła, ale w tym przypadku nie wydawało mi się to krytyczne.